src/main/resources/static/js/subscription.js

```diff
@@ -75,17 +75,24 @@ jQuery(document).ready(function () {
         if (backendStatus) {
             removeStatusIndicator();
         }
-        toggleButtonsDisabled(!backendStatus);
+        updateTable(backendStatus);
     }
 
     function toggleCheckboxesDisabled(disabled) {
         $('#check-all').prop("disabled", disabled);
         $('.data-check').prop("disabled", disabled);
     }
-
-    function toggleButtonsDisabled(disabled) {
-        $('.main #subButtons button.btn').prop("disabled", disabled);
-        $('.main #table button.btn').prop("disabled", disabled);
+    
+    var previousStatus;
+    function updateTable(currentStatus) {
+    	var statusChanged = (previousStatus != currentStatus);
+    	if(statusChanged && !currentStatus){
+    		table.clear().draw();
```

Comment:
Here it is tabs and not spaces, please fix.

Reply:
done